wandb/sdk/data_types.py

```diff
@@ -2121,6 +2234,112 @@ def from_json(
             masks=_masks,
         )
 
+    @classmethod
+    def from_rdkit(
```

Comment:
Under which conditions would the user choose to use the Image object instead of the Molecule object? As a matter of practicality, I think this will just change how the molecule is rendered in the UI. However, unlike Molecule, we lose information in the image generation (the actual molecule details are thrown away). I'm wondering if we should even have these two methods off of image?

Reply:
The conversion to 2D images would only work for smaller, simpler molecules. The original information should be preserved in the png metadata (see above). 
Do you think these two methods should be removed? How should we handle rendering in the UI? 
I see that certain capabilities are available through the `rdkit-js` package, see https://unpkg.com/@rdkit/rdkit@2021.9.1/Code/MinimalLib/dist/GettingStartedInJS.html